### DISCUSSION TITLE
Upload multiple images

### DISCUSSION THREAD

**Discussion by renja-g** [community/community#53235] [GitHub Education] [answered]
I have a problem, because my student ID has my name on the front and the date on the back,
and if I merge 2 images and upload them I get told to not upload them but to take a picture.

What can I do?

**Comment by samus-aran** [comment]
Hi @renja-grotemeyer 👋 

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status. We do accept multiple pieces of evidences so although you may have no date on the Student ID see if you can acquire additional evidence showing a valid date.

**Reply by renja-g** [reply]
But how would I upload multiple pictures/ images?
Or do I replay to the email?

**Reply by samus-aran** [reply]
The suggestion would be to take a photo of, as an example, a transcript with the Student ID on top of it. Just ensure that important information is seen. In this case, you had used a Student ID I would suggest to use another piece of evidence that isn't a Student ID.

**Reply by renja-g** [reply]
First what do you mean by "transcript ", also is it a problem if my student ID is German?
Because there is no "Date of Expiry" but the German translation.

**Reply by renja-g** [reply]
Because by now I'm at 22 tries, I keep getting the same error:
"Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 22, 2023 for..."

**Reply by renja-g** [reply]
@samus-aran Just to make sure because I don't know how the notifications work.

**Reply by samus-aran** [reply]
Hi @renja-grotemeyer if you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status. 

We can detect German but ensure your school name and profile information correspond with the translation as well when submitting a live photo. It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it.

I would suggest for your next submission to use a different type of evidence based on your recent rejection reasons.

**Reply by renja-g** [reply]
Hi @samus-aran first I wanted to say that I really appreciate that you are trying to help me, but no matter what I try I just can't get it:

**Till now, I tried:**

[<img alt="alt_text" width="10px" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5f/Red_X.svg/100px-Red_X.svg.png" />](https://www.google.com/) Dated school ID

[<img alt="alt_text" width="10px" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5f/Red_X.svg/100px-Red_X.svg.png" />](https://www.google.com/) Dated enrollment letter

[<img alt="alt_text" width="10px" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5f/Red_X.svg/100px-Red_X.svg.png" />](https://www.google.com/) Dated screenshot of school portal

[<img alt="alt_text" width="10px" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5f/Red_X.svg/100px-Red_X.svg.png" />](https://www.google.com/) Dated school ID + Dated enrollment letter

---

- My Profile name is the same as in the documents.
- I mentioned my school in my bio.
- I used my academic email address.

---

But no mater what I try I always get a mail with not real detail what I have to change just:
> "Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 24, 2023 for [@renja-grotemeyer](https://github.com/renja-grotemeyer)."

And I slowny run out of documents that I could try and even if I where to ask the shool for a custom certificat, I don't think it can get much clerer then my approved enrolment letter, which has my name, school name and dates on it.

**Reply by pandadaadaaa** [reply]
Yep. I have the same problem. I have tried my student ID, enrollment certificate, and student certificate, but none of them worked even though I have tried the English version of my certificate instead of the German version.

**Reply by renja-g** [reply]
sad to hear, let me know if you get it

**Reply by renja-g** [reply, marked as answer]
> Yep. I have the same problem. I have tried my student ID, enrollment certificate, and student certificate, but none of them worked even though I have tried the English version of my certificate instead of the German version.

I finally got accepted with just the enrolment letter, the only difference was that I was on the schools campus.

I hope this help.

**Reply by samus-aran** [reply]
That is very awesome to hear @renja-grotemeyer 👏  Thank you for coming back and giving us all an update 👍 

What benefit are you excited to try first? ⭐
